website/themes/hugo-geekdoc/layouts/partials/site-header.html

```diff
@@ -17,7 +17,7 @@
     </a>
 
     <span id="gdoc-dark-mode"> </span>
-    <a href="https://openbb.co/products/terminal" class="_btn w-full text-sm" target="_blank"
+    <a href="https://openbb.co/products/terminal" class="_btn" target="_blank"
```

Comment:
i forgot one quick thing sorry
because we are using target="_blank" to open the link in a new tab we also need this in a tag:
`rel="noopener noreferrer"`